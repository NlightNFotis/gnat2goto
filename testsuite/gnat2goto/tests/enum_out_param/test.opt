ALL XFAIL gnat2goto fails with "raised SYSTEM.ASSERTIONS.ASSERT_FAILURE : failed precondition from ireps.ads:1875"
