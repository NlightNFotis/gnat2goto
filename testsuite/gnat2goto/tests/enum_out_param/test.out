[enum_out.assertion.1] line 13 assertion : SUCCESS
VERIFICATION SUCCESSFUL
